docs/esa/neocc.rst

```diff
@@ -0,0 +1,327 @@
+
+.. _astroquery.esa.neocc:
+
+************************************************************************************************
+ESA NEOCC Portal Python Interface Library (`astroquery.esa.neocc`/`astroquery.solarsystem.neocc`)
+************************************************************************************************
+
+The ESA NEO Coordination Centre (NEOCC) is the operational centre of ESA’s Planetary Defence Office
+PDO) within the Space Safety Programme (S2P). Its aim is to coordinate and contribute to the
+observation of small Solar System bodies in order to evaluate and monitor the threat coming from 
+near-Earth objects (NEOs).
+
+ESA NEOCC Portal Python interface library makes the data that `ESA NEOCC <https://neo.ssa.esa.int/>`_
+provides easily accessible through a Python program.
+
+The main functionality of this library is to allow a programmer to easily retrieve:
+
+* All the NEAs
+* Other data that the NEOCC provides (risk list, close approach list, etc.)
+* All basic and advanced information regarding a NEA
+* An ephemeris service for NEAs
+
+==============================
+Getting ESA NEOCC's products
+==============================
+
+--------------------------------
+1. Direct download of list files
+--------------------------------
+This function allows the user to download the requested list data from ESA NEOCC.
+Different lists that can be requested are:
```

Comment:
you may need an empty line after the headings. RTD is failing currently, so I cannot double-check the rendering.

Reply:
Done